java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/ObjectAdapter.java

```diff
@@ -213,8 +209,8 @@ public void waitForHold() {
      */
     public void deactivate() {
         synchronized (this) {
-            // Wait for activation or a previous deactivation to complete.
-            // This is necessary to avoid out of order locator updates.
```

Comment:
And even though I expect this isn't a shared opinion, IMO, I think trying to maximize the length of lines in every possible case is counter-productive. I personally think comments are easier to read when the end-of-a-line corresponds to the end of a sentence or a natural break in the sentence.

I'm not saying we should fix the existing comments to do this, but as that one weirdo who goes out of his way to format comments for maximal readability, this sometimes means intentionally ending a line well-before I hit the column limit. And I'd prefer not for this intentional choice to be erased, just to maximize how many columns we can fit things into.